### PR TITLE
Add utils namespace.

### PR DESCRIPTION
**Pull request by siarheishut**
Move utils in rm::utils namespace. Fix tests.

### CODE REVIEW

File: src/map_renderer.h [hunk 1]
```diff
@@ -20,16 +20,18 @@
 #include "sphere.h"
 
 namespace rm {
+using namespace rm::utils;
```
Comment (foxfrikses) [inline src/map_renderer.h:23]: ?

File: tests/bus_manager_test.cpp [hunk 2]
```diff
@@ -711,7 +713,7 @@ TEST(TestBusManager, TestFindRouteInfo) {
       EXPECT_EQ(want[i].has_value(), got.has_value()) << name;
       if (!got || !want[i]) continue;
 
-      EXPECT_EQ(want[i], got) << name;
+      EXPECT_EQ(want[i].value(), got.value()) << name;
```
Comment (foxfrikses) [inline tests/bus_manager_test.cpp:716]: Why?

File: src/map_renderer.h (hunk 1 above)
Comment (siarheishut) [reply src/map_renderer.h]: Removed.

File: tests/bus_manager_test.cpp (hunk 2 above)
Comment (siarheishut) [reply tests/bus_manager_test.cpp]: Removed.

**Review by foxfrikses — APPROVED** [review]
LGTM

**Pull request merged**